### PR TITLE
i18n(es): update `i18n`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updates `i18n` based on this commit ee0cd38a1fae31717fe820e779baeabe693cd67a

### CODE REVIEW

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 1]
```diff
@@ -67,6 +67,8 @@ Starlight proporciona soporte incorporado para sitios multilingües, incluidas l
 
 </Steps>
 
+Para escenarios de i18n más avanzados, Starlight también admite la configuración de internacionalizaciónutlizando la [opción `i18n` de Astro](https://docs.astro.build/es/guides/internationalization/#configure-i18n-routing).
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/i18n.mdx:70]:
```suggestion
Para escenarios de i18n más avanzados, Starlight también admite la configuración de internacionalizaciónutlizando la [opción `i18n` de Astro](https://docs.astro.build/es/guides/internationalization/#configuración-de-rutas-i18n).
```

File: docs/src/content/docs/es/guides/i18n.mdx [hunk 2]
```diff
@@ -273,3 +275,15 @@ export const collections = {
...
+
+Puedes usar [`Astro.currentLocale`](https://docs.astro.build/es/reference/api-reference/#astrocurrentlocale) para leer el idioma actual en componentes `.astro`.
+
+El siguiente ejemplo lee el idioma actual y lo utiliza para generar un enlace a una página Acerca de en el idioma actual:
+
+```astro
+---
+// src/components/AboutLink.astro
+---
+<a href={`/${Astro.currentLocale}/about`}>About</a>
```
Comment (HiDeoo) [inline docs/src/content/docs/es/guides/i18n.mdx:289]: Would it make sense to translate the link "About" text?

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and except for 1 small change, this looks good to me.

File: docs/src/content/docs/es/guides/i18n.mdx (hunk 2 above)
Comment (dreyfus92) [inline docs/src/content/docs/es/guides/i18n.mdx:289]:
```suggestion
<a href={`/${Astro.currentLocale}/about`}>Acerca</a>
```

**Review by HiDeoo — APPROVED** [review]
Thanks for updating the PR :thumbsup: Looks perfect to me.

**Pull request merged**